### PR TITLE
Flush stdout

### PR DESCRIPTION
**Pull request by jbenito3**

### CODE REVIEW

File: indico_mattermost_bot.py [hunk 1]
```diff
@@ -68,6 +73,9 @@ def _process_bots(config):
     return bots, channel_hooks
 
 
+def _is_fetching_past_events(bot):
+    return bot['timedelta'].startswith('-')
+
```
Comment (pferreir) [inline indico_mattermost_bot.py:79]: Doesn't PEP8 require an additional line break here?

File: indico_mattermost_bot.py [hunk 2]
```diff
@@ -15,6 +16,10 @@
 notified = set()
 
 
+def _info(message):
+    print(message)
+    sys.stdout.flush()
+
```
Comment (pferreir) [inline indico_mattermost_bot.py:22]: PEP8 again?

Comment (jbenito3) [reply indico_mattermost_bot.py:22]: Yes...you are right...modifying...

**Comment by pferreir** [comment]
LGTM 👍

**Pull request merged**